Tools/msg/generate_msg_docs.py

```diff
@@ -2,13 +2,105 @@
 
 """
 Generate docs from .msg files
+Also generates docs/en/middleware/dds_topics.md from dds_topics.yaml
 """
 
 import os
 import argparse
 import sys
 
 
+import yaml
+
+def generate_dds_yaml_doc(allMessageFiles, output_file = 'dds_topics.md'):
```

Comment:
Note, I essentially hard code the output to the PX4 docs tree. 

If this needs to be done differently I could add to build tree, then copy could happen separately - i.e.like the uorb topics.